src/main/kotlin/FinalProject/Car.kt

```diff
@@ -0,0 +1,29 @@
+package FinalProject
+
+/* class for a single car in the simulation
+*/
+class Car(d: Boolean) {
+	/* variables
+	 */
+	private var drive = d
```

Comment:
The desire to drive does not change, so why not make this a `val` - prevents accidentally creating a bug.